rust_src/src/keymap.rs

```diff
@@ -317,6 +317,93 @@ pub fn map_keymap_lisp(function: LispObject, keymap: LispObject, sort_first: boo
     LispObject::constant_nil()
 }
 
+/// Call FUN for every binding in MAP and stop at (and return) the parent.
+/// FUN is called with 4 arguments: FUN (KEY, BINDING, ARGS, DATA).  */
```

Comment:
You forgot to remove the `*/` at the end of the comment.

Reply:
Whoops. I should make an elisp function to deal with conversion of comments. Thanks will fix.